firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/model/Architecture.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.crashlytics.internal.model;
+
+import android.os.Build;
+import android.text.TextUtils;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public enum Architecture {
```

Comment:
Android style is generally to use `IntDef`s instead of `enum`s, right? Personally, I prefer enums, but thought I'd point it out. If it were part of the public API, it would be a bigger deal.

Reply:
Hmm, I guess it depends on which camp you're in (perfmatters/enummatters). :stuck_out_tongue_closed_eyes: But in this case I think an `IntDef` actually makes more sense.